doc/libpmempool/pmempool_feature_query.3.md

```diff
@@ -0,0 +1,108 @@
+[comment]: <> (are met:)
+[comment]: <> (    * Redistributions of source code must retain the above copyright)
+[comment]: <> (      notice, this list of conditions and the following disclaimer.)
+[comment]: <> (    * Redistributions in binary form must reproduce the above copyright)
+[comment]: <> (      notice, this list of conditions and the following disclaimer in)
+[comment]: <> (      the documentation and/or other materials provided with the)
+[comment]: <> (      distribution.)
+[comment]: <> (    * Neither the name of the copyright holder nor the names of its)
+[comment]: <> (      contributors may be used to endorse or promote products derived)
+[comment]: <> (      from this software without specific prior written permission.)
+
+[comment]: <> (THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS)
+[comment]: <> ("AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT)
+[comment]: <> (LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR)
+[comment]: <> (A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT)
+[comment]: <> (OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,)
+[comment]: <> (SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT)
+[comment]: <> (LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,)
+[comment]: <> (DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY)
+[comment]: <> (THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT)
+[comment]: <> ((INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE)
+[comment]: <> (OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.)
+
+[comment]: <> (pmempool_feature_query.3 -- man page for toggle and query pool
+set features)
+
+[NAME](#name)<br />
+[SYNOPSIS](#synopsis)<br />
+[DESCRIPTION](#description)<br />
+[RETURN VALUE](#return-value)<br />
+[SEE ALSO](#see-also)<br />
+
+
+# NAME #
+
+_UW(pmempool_feature_query), _UW(pmempool_feature_enable),
+_UW(pmempool_feature_disable) - toggle or query pool set features
+
+
+# SYNOPSIS #
+
+```c
+#include <libpmempool.h>
+
+_UWFUNCR1(int, pmempool_feature_query, *path, enum pmempool_feature feature)
+
+_UWFUNCR1(int, pmempool_feature_enable, *path, enum pmempool_feature feature)
+
+_UWFUNCR1(int, pmempool_feature_disable, *path, enum pmempool_feature feature)
+```
+
+_UNICODE()
+
+
+# DESCRIPTION #
+
+The *feature* argument accepts following values:
+
++ **PMEMPOOL_FEAT_SINGLEHDR** - only the first part in each replica contains the
+pool part internal metadata. This value can be used only with
+**pmempool_feature_query**(). It can not be enabled or disabled. For details see
+**poolset**(5).
+
++ **PMEMPOOL_FEAT_CKSUM_2K** - only the first 2KiB of pool part internal metadata
+is checksummed. It is required to use other features e.g.
+**PMEMPOOL_FEAT_SHUTDOWN_STATE**. When **PMEMPOOL_FEAT_CKSUM_2K** is disabled
+whole 4KiB is checksummed.
```

Comment:
From reading this description the dependency between shutdown_state and cksum_2k is not clear to me.

There's also a question of whether user is supposed to enable this feature manually when he/she wants stutdown_state or it's going to happen automatically.